parsing/lexer.mll

```diff
@@ -382,9 +384,9 @@ rule token = parse
       { UNDERSCORE }
   | "~"
       { TILDE }
-  | ".~"
-      { error lexbuf
-          (Reserved_sequence (".~", Some "is reserved for use in MetaOCaml")) }
+  | ".<" { DOTLESS }     (* NNN *)
+  | ">." { GREATERDOT }  (* NNN *)
```

Comment:
It appears that this will break existing code that uses `>.` as an operator, such as [this](https://github.com/janestreet/core/blob/36053c38f9cfb7391b1f7dd37231861700e3599e/core/src/robustly_comparable.ml#L14):

```ocaml
val (>.) : t -> t -> bool
```